lib/metadata_listener/fits_service.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'http'
 require 'nokogiri'
```

Comment:
we should be able to remove this, and also from the Gemfile.

Reply:
Beat me to it... I saw the new commit that removes it. Try taking it out of the Gemfile. I don't think we're using it anywhere else.